code/modules/hydroponics/hydro_tray.dm

```diff
@@ -619,6 +619,59 @@
 	if(..())
 		return 1
 
+	if(istype(O, /obj/item/claypot))
+		user << "<span class='warning'>You must place the pot on the ground and use a spade on \the [src] to make a transplant.</span>"
+		return
+
+	if(seed && istype(O, /obj/item/weapon/pickaxe/shovel/spade))
```

Comment:
@Probe1 I'd say trays used in botany are a little too delicate for a huge shovel used by mining.